FrontEnd/src/components/ProfilePage/FormComponents/AdditionalInfo.js

```diff
@@ -42,74 +26,73 @@ const AdditionalInfo = (props) => {
+                const response = await fetch(`${process.env.REACT_APP_BASE_API_URL}/api/profiles/${user.profile_id}`, {
+                    method: 'PATCH',
+                    headers: {
+                        'Authorization': `Token ${token}`,
+                        'Content-Type': 'application/json',
+                    },
+                    body: JSON.stringify({
+                        founded: profile.founded,
+                    }),
+                });
+
+                if (response.status === 200) {
+                    const updatedProfileData = await response.json();
+                    profileMutate(updatedProfileData);
+                } else {
+                    console.error('Помилка');
+                }
+            } catch (error) {
+                console.error('Помилка:', error);
+            }
         }
     };
 
-    return (
-        <div className={css['form__container']}>
-            <form id="AdditionalInfo" onSubmit={handleSubmit} autoComplete="off" noValidate>
-                <div className={css['fields']}>
-                    <div className={css['fields-groups']}>
-                        <HalfFormField
-                            inputType="number"
-                            name="foundationYear"
-                            label={LABELS.foundationYear}
-                            updateHandler={onUpdateFoundationYearField}
-                            requredField={false}
-                            value={user.foundationYear}
-                            error={foundationYearError}
-                        />
-                        <HalfFormField
-                            inputType="number"
-                            name="companySize"
-                            label={LABELS.companySize}
-                            updateHandler={onUpdateField}
-                            requredField={false}
-                            value={user.companySize}
-                        />
-                    </div>
-                    <TextField
-                        name="topClients"
-                        label={LABELS.topClients}
-                        updateHandler={onUpdateTextAreaField}
-                        requredField={false}
-                        value={user.topClients}
-                        maxLength={TEXT_AREA_MAX_LENGTH}
-                    />
-                    <TextField
-                        name="passedAudit"
-                        label={LABELS.passedAudit}
-                        updateHandler={onUpdateTextAreaField}
-                        requredField={false}
-                        value={user.passedAudit}
-                        maxLength={TEXT_AREA_MAX_LENGTH}
-                    />
-                </div>
-            </form>
-        </div>
-    );
-};
+        return (
+            <div className={css['form__container']}>
+                {(user && profile && mainProfile)
+                    ?
+                    <>
```

Comment:
No need for the Fragment since `form` only one element

Reply:
changed in all similar cases in other files